android/build.gradle

```diff
@@ -62,14 +62,18 @@ android {
 }
 
 repositories {
+  maven {
+    // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
+    url "$rootDir/../node_modules/react-native/android"
+  }
```

Comment:
If you have a project XYZ, it will declare its own repositories that are used to resolve dependencies. This is typically done in an `allprojects` block (which will then also be applied to all Gradle sub-projects). You can see how it's done in the official template:
https://github.com/facebook/react-native/blob/master/template/android/build.gradle#L22
Again, if you think there is a real problem, please point me to a sample project that has this problem, and I will take a look as soon as I can.
Meanwhile, if you follow the test plan instructions above, you will see that it **will fix the issue** in this repo. If other follow-up changes are required, we can do them in a separate PR.